controller/src/controller/db/client.py

```diff
@@ -53,8 +47,10 @@ def _from_db_object(obj, obj_class):
     def _merge_into_db_object(obj, obj_class):
         """
         Merge a schema object into an DB object.
```

Comment:
can you explain what it means to merge? if I wanted to implement this abstract method, I wouldn't know what to do here. Abstract methods require more information so their implementer will know what is expected from them.

Reply:
irrelevant, not abstract anymore